srtcore/buffer_rcv.h

```diff
@@ -67,6 +67,11 @@ class CRcvBufferNew
     // TODO: Previously '-2' also meant 'already acknowledged'. Check usage of this value.
     int insert(CUnit* unit);
 
+    /// @brief Drop a unit from the buffer.
+    /// @param pos position of the unit to drop.
+    /// @return false if nothing to drop, true if the unit was dropped successfully.
+    bool dropUnit(int pos);
+
```

Comment:
- [ ] The function has to be private, as it operates on the knowledge of the internal position of the unit in the array of entries.

```suggestion
    /// @brief Drop a unit from the buffer.
    /// @param pos position in the m_entries of the unit to drop.
    /// @return false if nothing to drop, true if the unit was dropped successfully.
    bool dropUnit(int pos);

```